assets/js/components/settings/settings-admin.js

```diff
@@ -21,18 +21,50 @@
  */
 import { Fragment } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
 import Layout from '../layout/layout';
+import { Cell } from '../../material-components';
 import OptIn from '../optin';
 import VisuallyHidden from '../VisuallyHidden';
 import ResetButton from '../ResetButton';
+import UserInputPreview from '../user-input/UserInputPreview';
+import { USER_INPUT_QUESTIONS_LIST } from '../user-input/util/constants';
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import UserInputSettings from '../notifications/UserInputSettings';
+const { useSelect } = Data;
 
 const SettingsAdmin = () => {
+	const isUserInputCompleted = useSelect( ( select ) => select( CORE_USER ).getUserInputState() === 'completed' );
+	const userInputURL = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-user-input' ) );
+
+	const goTo = ( questionIndex = 1 ) => {
+		global.location.assign( addQueryArgs( userInputURL, {
+			question: USER_INPUT_QUESTIONS_LIST[ questionIndex - 1 ],
+			redirect_url: global.location.href,
+		} ) );
+	};
+
 	return (
 		<Fragment>
+			{ featureFlags.userInput.enabled && (
+				<Cell size={ 12 }>
+					<Layout>
+						{ isUserInputCompleted ? (
+							<div className="googlesitekit-module-page googlesitekit-settings-user-input">
+								<UserInputPreview goTo={ goTo } noFooter />
+							</div>
+						) : (
+							<UserInputSettings isDimissable={ false } />
+						) }
```

Comment:
In the future I'd prefer we not use ternaries in JSX and use something like:

```suggestion
						{ isUserInputCompleted && (
							<div className="googlesitekit-module-page googlesitekit-settings-user-input">
								<UserInputPreview goTo={ goTo } noFooter />
							</div>
						) }
						{ ! isUserInputCompleted && <UserInputSettings isDimissable={ false } /> }
```